### ISSUE TITLE
Doesn't work for Kali with Thunderbolt cable?

### ISSUE THREAD

**Issue by therealjayvi90** [carmaa/inception#129]
I see all over the place that this is supposed to work over Thunderbolt cable, but I haven't been able  to actually get it to register that anything is even plugged in. Is there something I'm missing? There seems to be no Interface for Thunderbolt...

**Comment by carmaa** [comment]
Assuming you are using Kali as the attack platform? I haven't tested thunderbolt support for Linux yet, so not sure that will work. Thunderbolt from a MacOS works.

**Comment by therealjayvi90** [comment]
Hey thanks for answering! So I dual boot Kali on a newer MacBook pro but I wasn't able to get it to work on the Kali side. I've finally got inception working properly from my MacOS side of things, though whenever I run the tool the only "available modules" found when I dry-run the command are "file, firewire, slotscreamer". Should I just try to use an older Mac with direct firewire access as my host machine instead?

**Comment by therealjayvi90** [comment]
I'm at my wits end at this point... I purchased a Thunderbolt to Firewire adapter, then realized I didn't have a 1394 9-pin to 4-pin cable so I bought one of those as well. Woke up this morning with everything connected and still inception doesn't show thunderbolt as an option; calling inception with firewire says that no interfaces have been found. Am I going about this incorrectly?

**Comment by therealjayvi90** [comment]
Managed to figure out the issue. Newer MacBooks are restricted to only Thunderbolt ports as an option, and newer versions of MacOS (>= El Capitan) have removed /dev/fw* as an option while using Thunderbolt; even with using any type of conversion adapter. 
Booting into Kali I noticed that by using a Thunderbolt to iEEE 9-pin adapter, combined with a 9-pin to 4-pin cable, I had devices showing as /dev/fw0 and /dev/fw1. Worked perfectly from this point on!

For reference I'm using a 2015 15" MacBook Pro i7, dual-booted to the latest release of Kali (amd64).
